src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -138,6 +158,12 @@ void TilePyramid::update(const std::vector<Immutable<style::Layer::Impl>>& layer
     };
 
     renderTiles.clear();
+
+    if (!panTiles.empty()) {
+        algorithm::updateRenderables(getTileFn, createTileFn, retainTileFn,
+                [](const UnwrappedTileID&, Tile&) {}, panTiles, zoomRange, panZoom);
+    }
+
```

Comment:
Capturing from the chat: we have a separated `algorithm::updateRenderables` because we don't create render tiles for pan tiles.